.github/workflows/ci.yml

```diff
@@ -9,7 +9,9 @@ on:
   workflow_dispatch:
 
 env:
+  MAVEN_ARGS: "-B -nsu -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=120"
   SUREFIRE_RERUN_FAILING_COUNT: 2
```

Comment:
Is this variable still necessary?

Reply:
Seems so, not sure how it's used but it is used here:
https://github.com/keycloak/keycloak/blob/08dfdffbfbea8ea8b4191cec5ee1926b2a386c99/.github/workflows/ci.yml#L394